conjur/wrapper/http_wrapper.py

```diff
@@ -36,6 +37,7 @@ def invoke_endpoint(http_verb, endpoint, params, *args, check_errors=True,
     """
     This method flexibly invokes HTTP calls from 'requests' module
     """
+    urllib3.disable_warnings()
```

Comment:
Is it OK to disable it permanently?

Reply:
we disable the ugliness and replace it with a more readable error that open shows up on init. See my description [here](https://github.com/cyberark/conjur-api-python3/pull/253/files#r605500983). WDYT?